index.html

```diff
@@ -1797,23 +1775,22 @@
                 </li>
                 <li>
                   <a>Queue a task</a> to <a>fire</a> an event named
-                  <code>connectionavailable</code> at <a data-lt=
-                  "PresentationReceiver"><code>presentation.receiver</code></a>.
+                  <code>connectionavailable</code> at all
```

Comment:
So the step to "fire an event named connectionavailable at all PresentationConnectionList" should only target the objects that are associated with the receiving browsing context in this instance of the algorithm, not every object in the receiving user agent.   Just saying "...at the PresentationConnectionList associated with the receiving browsing context" would be fine for now.